src/components/Header.js

```diff
@@ -1,25 +1,27 @@
-import React from "react"
+/** eslint-disable */
```

Comment:
As liners are guidance for best practice Linters shouldn't be disabled. Kindly fix it.